cram_environment_representation/src/object-perceptions.lisp

```diff
@@ -51,7 +51,17 @@ just updated. Otherwise a new instance is created."))
 (defmethod register-object-designator-data
     ((data cram-manipulation-knowledge:object-shape-data-mixin) &key type)
   (declare (ignore type))
-  nil)
+  (let* ((object (event-object-designator data))
```

Comment:
There is no `event-object-designator` slot in `object-shape-data-mixin`. Is there a pull request on cram_highlevel missing?